configure.ac

```diff
@@ -6231,6 +6264,14 @@ fi
 
 if test "x$with_libdpdk" = "xyes"
 then
+  plugin_dpdkevents="no (DPDK version < 16.07)"
```

Comment:
You mean applying it along with AC_COMPILE_IFELSE, right ? Seems reasonable, I'll  do it.  Thanks